packages/wdio-repl/tests/repl.test.ts

```diff
@@ -1,10 +1,15 @@
 import vm from 'vm'
 import replMock from 'repl'
+import expect from 'expect'
```

Comment:
Any idea how I can get rid of this? Without this TypeScript believes we use `expect-webdriverio`. The problem importing `expect` is that it would be redundant and Jest would mock dependencies away that we have defined in this project.

Reply:
I feel like we should exclude the type in that case since apparently it is installed I think?

Reply:
How would you exclude it?